source/ed/osm2ed_main.cpp

```diff
@@ -0,0 +1,35 @@
+/* Copyright © 2001-2018, Canal TP and/or its affiliates. All rights reserved.
+
+This file is part of Navitia,
+    the software to build cool stuff with public transport.
+
+Hope you'll enjoy and contribute to this project,
+    powered by Canal TP (www.canaltp.fr).
+Help us simplify mobility and open public transport:
+    a non ending quest to the responsive locomotion way of traveling!
+
+LICENCE: This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+Stay tuned using
+twitter @navitia 
+IRC #navitia on freenode
+https://groups.google.com/d/forum/navitia
+www.navitia.io
+*/
+
+#include "osm2ed.h"
+
+int main(int argc, const char** argv) {
+    return ed::connectors::osm2ed(argc, argv);
+}
```

Comment:
newline plz :)

Reply:
Added in cc7a068320fa0493468adccc2f634394c667a370 :)
But are we not using C++11 feature set ? I thought new lines were not a requirement anymore for end of files.

Reply:
We are, but it's more elegant when github doesn't warn :)

Reply:
got it :+1: